src/lang/export.pl

```diff
@@ -44,3 +44,28 @@
   tripledb_export(Directory),
   obda_export(Directory),
   forall(memorize_hook(Directory), true).
+
+     /*******************************
+     *          UNIT TESTS          *
+     *******************************/
+
+:- begin_tests('lang_export').
+
+get_path(Path):-
+  working_directory(X,X), string_concat(X, "test_lang_export", Path).
+
+test('stores knowledge in test_lang_export directory', 
+  [ setup(get_path(Path))
```

Comment:
the error here indicates that the tf_plugin was not loaded properly but that the export hook expects it to be loaded